src/test/java/tests/TurboMilestoneTest.java

```diff
@@ -34,6 +32,45 @@ public void turboMilestoneTest() {
     }
 
     @Test
+    public void getDueDateComparator_combinationOfDifferentMilestones_correctMilestoneOrdering() {
```

Comment:
The naming convention is (note that 2nd and 3rd parts are Pascal case so that the whole thing remains camelCase):

**`MethodName_StateUnderTest_ExpectedBehavior`** 

Following are some of the example:
`isAdult_AgeLessThan18_False`
`withdrawMoney_InvalidAccount_ExceptionThrown`
`admitStudent_MissingMandatoryFields_FailToAdmit`